### DISCUSSION TITLE
X the Solar PV1 or PV2 current or voltage in entities will break the card

### DISCUSSION THREAD

**Discussion by deberha2** [slipx06/sunsynk-power-flow-card#219] [Q&A] [answered]
I am using this card with my Sol-Ark12K.  The card is great!  I linked PV1 and PV2 to the watts output from each of my two AC-coupled Enphase arrays.  I do not have any solar panels connected directly to the Sol-Ark.  That was working.  Then I wanted to try to remove the current and voltage display from PV1 and PV2 on the card since I don't really need those.  When I "x" all 4 of those and tried to save them as blank, I broke the card.  I opened a new card and can find the values that need to be put back on those solar entities (IE "sensor.sunsynk_pv1_voltage", but I don't know where or how to do that.  I'm not a programmer but I can usually figure things out...  The entities in question are: pv1_voltage_109 , pv1_current_110 , pv2_voltage_111 , pv2_current_112.
The error that is shown is "! Please include the day_pv_energy_108 attribute and entity ID" and the card is no longer displayed.

**Comment by slipx06** [comment, marked as answer]
Hi. Its looking for the entity `day_pv_energy_108: sensor.sunsynk_day_pv_energy`.  You should add it back to your config. You can set it to none if you don't have a sensor for this. 

```
entities:
  day_pv_energy_108: none
```

You can delete all of these from your config to hide them

 ```
  pv1_voltage_109: sensor.sunsynk_pv1_voltage
  pv1_current_110: sensor.sunsynk_pv1_current
  pv2_voltage_111: sensor.sunsynk_pv2_voltage
  pv2_current_112: sensor.sunsynk_pv2_current
```
